packages/manager/modules/web-paas/src/details/service/translations/Messages_es_ES.json

```diff
@@ -37,5 +37,6 @@
   "web_paas_expiry_date": "Fecha de expiración",
   "web_paas_tile_subscription_manage_service": "Gestionar el servicio",
   "web_paas_tile_subscription_pay_billing": "Pagar la factura",
-  "web_paas_tile_billing_error": "Se ha producido un error al redirigir la página a la facturación: "
+  "web_paas_tile_billing_error": "Se ha producido un error al redirigir la página a la facturación: ",
+  "web_paas_service_project_coming_soon": "Próximamente disponible"
```

Comment:
where does those translations comes from ? i don't see the translator commit, did you add them manually ?

Reply:
@frenautvh 
We had this translation already done but i lost it when i rebased. So i added it manually.

Reply:
@radireddy, Please check now, it looks clean and i have imported the new translation changes.